packages/trader/src/Stores/Modules/Trading/Helpers/limit-orders.ts

```diff
@@ -53,7 +52,6 @@ export const setLimitOrderBarriers = ({
 
                 barrier.onChange({
                     high: obj_limit_order.value,
-                    low: undefined, //TODO: wait until ChartBarrierStore is ts migrated and 'low' can be an optional parameter
```

Comment:
Regarding line 88:
`ReturnType<typeof useStore>['contract_trade']['contracts'][number]`
Will it work? Asking because in 4th package its been used